web/app-vtp.properties

```diff
@@ -17,6 +17,7 @@ k9.psb.uttak.url=http://localhost:8686/pleiepenger-barn-uttak
 saf.base.url=https://localhost:8063/rest/api/saf
 DOKDISTFORDELING_URL=http://localhost:8060/rest/dokdist/v1
 DOKARKIV_URL=http://localhost:8060/rest/dokarkiv/rest/journalpostapi/v1
+DOKARKIV_BASE_URL=https://vtp:8063/rest/dokarkiv
```

Comment:
Hvorfor er det to dokarkiv url-er på to forskjellige porter her?

Reply:
Eg tipper det er ulike porter for journalføring og henting av journalposter